demo/demo/tests/test_apple_signed_payload_validation.py

```diff
@@ -140,7 +140,7 @@ def test__apple_root_certificate_not_set_up():
     import subscriptions.providers.apple_in_app.app_store
     subscriptions.providers.apple_in_app.app_store.get_original_apple_certificate.cache_clear()
     with mock.patch('subscriptions.providers.apple_in_app.app_store.DEFAULT_ROOT_CERTIFICATE_PATH',
-                    return_value=pathlib.Path('./dummy.cer')):
+                    new=pathlib.Path('./dummy.cer')):
```

Comment:
This is a weird test overall, why do you check what happens if default path is not a default path? It is a bit equal to testing that app breaks if you install Flask instead of Django.